example/src/containers/MetaMaskConnector/MetaMaskConnector.tsx

```
@@ -21,15 +21,9 @@ export const MetaMaskConnector = () => {
     const installSnap = useCallback(async () => {
        const isInitiated = await installPolkadotSnap();
        if(!isInitiated) {
-          dispatch({type: MetamaskActions.SET_INSTALLED_STATUS, payload: {isInstalled: false, message: "Please accept snap installation prompt"}})
+        dispatch({type: MetamaskActions.SET_INSTALLED_STATUS, payload: {isInstalled: false, message: "Please accept snap installation prompt"}})
        } else {
-          dispatch({type: MetamaskActions.SET_INSTALLED_STATUS, payload: {isInstalled: true}});
-
-           const dotAssetAdded = await addDotAsset();
-           if (!dotAssetAdded) {
-               dispatch({type: MetamaskActions.SET_INSTALLED_STATUS, payload: {isInstalled: false, message: "Failed to add dot asset to metamask"}})
-           }
-           addDotAsset().catch(() => alert("Failed to add dot asset"));
+           addDotAsset().catch(() => dispatch({type: MetamaskActions.SET_INSTALLED_STATUS, payload: {isInstalled: false, message: "Failed to add dot asset to metamask"}}));
```

I think we can let this silently fail